src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,8 +5,6 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
-    public Excavator() {
-    }
```

Comment:
* Add some custom fields that could be only in Excavator
 * Do not remove no-args constructor

Reply:
If class doesn't have any other constructors - compiler adds constructor by default automatically